spec/Powertrain/EngineCoolant.vspec

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2016 Contributors to COVESA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+
+Capacity:
+  datatype: float
+  type: attribute
+  description: Engine coolant capacity in liters.
+  unit: l
+
+Temperature:
+  datatype: float
+  type: sensor
+  unit: celsius
+  description: Engine coolant temperature.
+
+Level:
+  datatype: string
+  type: sensor
+  allowed: [
+    'CRITICALLY_LOW',  # Critically low, immediate action required
+    'LOW',             # Level below recommended range, but not critical
+    'NORMAL',          # Within normal range, no need for driver action
```

Comment:
Should this also have a HIGH and CRITICALLY_HIGH level as for EngineOil.Level?

Reply:
It would at least not hurt. I actually do not know what sensors modern high-end vehicles have for coolant; like can they detect and report that there is too much coolant?

Reply:
I don't know enough about the mechanisms, but one possibility may be that the same kind of sensor could be used for both, for example. As well, what I am proposing seems to provide consistency that could be reused for other "level" sensors (washer fluid, transmission fluid, etc.). We do have another issue tackling common signals. Maybe we should consider a fluid-level signal that can be re-used for all these kinds of sensors?

Reply:
I did consider reusing this and have the same enums. Technically I don't know what a typical sensor for the coolant could do; From a functional perspective I don't know if it makes sense to have HIGH and CRITICALLY_HIGH. For the engine oil I can imagine that having the right amount between 2 thresholds is important. But with coolant I don't think there is an upper threshold (other than you'll probably get wet shoes).
ISO 7000 2429, which is meant for the coolant level telltale describes 'low' or 'out of specified values'. In that case it would maybe make more sense to have something like  'NORMAL', 'LOW', 'CRITICAL'.